lib/streams/compound-duplex.js

```diff
@@ -102,4 +109,37 @@ CompoundDuplex.prototype._abortStream = function() {
 	}
 };
 
+CompoundDuplex.prototype.wrap = function(stream) {
```

Comment:
I'm still not entirely convinces we should change the standard. There's already a [Readable.prototype.wrap](https://github.com/joyent/node/blob/master/lib/_stream_readable.js#L755). 
The method I went with was to extend this built in method to other types of streams.

I do agree that convenience streams built around these types have an extra method that doesn't make sense.